### PR TITLE
task 9349: create new wallet screen - create UI

### PR DESCRIPTION
**Pull request by vanpd-0859**
### Issue Link :link:

https://edu-redmine.sun-asterisk.vn/issues/9349
### Goals :soccer:

https://marvelapp.com/3feje8d/screen/54836925
- Wallet image + Label
- Wallet name textfield
- Password + Repeat password textfield
- Create button
### Implementation Details :construction:

Screen Shot 2019-04-02 at 10 09 09 AM

### CODE REVIEW

**Comment by vanpd-0859** [comment]
ready

File: CryptoWallet/Scenes/Login/CreateWallet/CreateWalletViewController.swift [hunk 1]
```diff
@@ -10,8 +10,22 @@ import UIKit
 import Reusable
 
 final class CreateWalletViewController: UIViewController {
+    @IBOutlet weak var walletNameTextField: UITextField!
```
Comment (longvt96) [inline CryptoWallet/Scenes/Login/CreateWallet/CreateWalletViewController.swift]: add private tương tự ở dưới

File: CryptoWallet/Extensions/UIView+.swift [hunk 2]
```diff
@@ -10,7 +10,7 @@ import Foundation
 import UIKit
 
 extension UIView {
-    func setBorder(cornerRadius: CGFloat = 0, borderWidth: CGFloat = 0, borderColor: UIColor) {
+    func setBorderRadius(cornerRadius: CGFloat = 0, borderWidth: CGFloat = 0, borderColor: UIColor) {
```
Comment (tienpx3010) [inline CryptoWallet/Extensions/UIView+.swift]:
```suggestion
    func setBorder(cornerRadius: CGFloat = 0, borderWidth: CGFloat = 0, borderColor: UIColor) {
```

File: CryptoWallet/Extensions/UITextField+.swift [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+//  Created by Phan Dinh Van on 4/2/19.
+//  Copyright © 2019 Phan Dinh Van. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+extension UITextField {
+    func underlined(height: CGFloat, color: UIColor) {
+        let underline = CALayer()
```
Comment (tienpx3010) [inline CryptoWallet/Extensions/UITextField+.swift]: Em có thể sử dụng .then cho gọn code

**Review by longvt96 — APPROVED** [review]

**Comment by vanpd-0859** [comment]
ready

**Comment by vanpd-0859** [comment]
ready

**Comment by vanpd-0859** [comment]
ready

**Review by tienpx3010 — APPROVED** [review]

**Pull request merged**